tilings/strategies/experimental_verification.py

```diff
@@ -0,0 +1,125 @@
+"""
+This file contains "experimental" verification strategies, which are those that verify
+tilings for which we are not certain we can independently calculate their counting
+sequence or generating function.
+"""
+
+from typing import Iterable, Iterator, Optional
+
+from comb_spec_searcher import StrategyFactory, VerificationStrategy
+from comb_spec_searcher.strategies import VerificationRule
+from permuta import Av, Perm
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import Factor, SubclassVerificationAlgorithm
+
+__all__ = [
+    "SubclassVerificationFactory",
+]
+
+TileScopeVerificationStrategy = VerificationStrategy[Tiling, GriddedPerm]
+
+
+class SubclassVerificationStrategy(TileScopeVerificationStrategy):
+    """
+    A strategy object for holding the results of SubclassVerification
+    """
+
+    def __init__(
+        self, subclass_basis=Iterable[Perm], ignore_parent: bool = True,
+    ):
+        self.subclass_basis = tuple(sorted(subclass_basis))
+        super().__init__(ignore_parent=ignore_parent)
+
+    def verified(self, tiling: Tiling) -> bool:
+        algo = SubclassVerificationAlgorithm(tiling, set(self.subclass_basis))
+        return algo.subclasses == self.subclass_basis
```

Comment:
thought we said it should be algo.is_verified()

Reply:
I realise you need all patterns to be in perm left to guarantee you avoid all of them. Ignore me!